### ISSUE TITLE
Guidance needed on adding data in 'My Data': which repositories can WT connect to?

### ISSUE THREAD

**Issue by benmarwick** [whole-tale/dashboard#188]
I've been trying to add data to 'My Data' via DOIs, but I cannot get anything to work from:

- figshare
- zenodo
- open science framework (OSF)
- the digital archaeological record (tDAR)

It has been a bit frustrating to find that none of those repositories work with WT. 

Perhaps some documentation could be added to the 'My Data' page to tell the user what repositories _are_ available to connect to via DOI? Or a link to a list of repositories that WT _does_ work with? 

A similar request was made here: https://github.com/whole-tale/whole-tale/issues/8 And this more recent issue seems to note a similar need for user-facing information about how to get data into the  Tale: https://github.com/whole-tale/dashboard/issues/190

**Comment by ThomasThelen** [comment]
Thanks for the feedback @benmarwick! We currently don't support data registration from figshare, zenodo, or OSF. Sorry about any frustration-I added additional information to the [documentation](http://wholetale.readthedocs.io/users_guide/manage.html) about how to register data and where data can be registered from.

While investigating the tDAR registration issue I noticed that the DOIs in DataONE look incorrect for some number of the tDAR packages (I didn't test them all). For example,
https://search.dataone.org/#view/doi:10.6067:XCV8NS0XGV_meta$v=1529519441290
I'll look into this and file it with the party responsible and that should hopefully clear up the issue of tDAR packages not being found.
